forge-ai/src/main/java/forge/ai/AiController.java

```diff
@@ -542,6 +542,8 @@ private Card chooseBestLandToPlay(CardCollection landList) {
         }
 
         // try to skip lands that enter the battlefield tapped if we might want to play something this turn
+        // TODO figure out a better life level for shocking in lands
+        boolean check_shocks = player.getLife() > 10;
```

Comment:
unused variable?

Reply:
Yep, good catch! A side effect of coding past midnight 😅